package.json

```diff
@@ -7,9 +7,6 @@
       "packages/terriajs-server",
       "packages/plugin-api",
       "packages/plugin-sample"
-    ],
-    "nohoist": [
```

Comment:
This has moved to the `nmHoistingLimits` in `yarnrc.yml`. Someone who uses workspaces should test that it still works as intended.